_includes/post.html

```diff
@@ -0,0 +1,64 @@
+{% assign date_format = site.minima.date_format | default: "%b %-d, %Y" %}
+
+
+{% for category in site.categories %}
```

Comment:
I did not mean the for loop in this file, just the part to display a post, i.e. starting on line 16